com.unity.render-pipelines.high-definition/Documentation~/Mask-Map-and-Detail-Map.md

```diff
@@ -7,7 +7,12 @@ This document describes the format of the mask map and detail map so that you ca
 To create a mask map:
```

Comment:
I'd state here that this example uses Adobe Photoshop (so we can be more explicit) but you can use any image editing someware that supports channel editing. Then remove the first instruction.